### ISSUE TITLE
Handle crawler response status codes that indicate failure

### ISSUE THREAD

**Issue by mgnoonan** [mgnoonan/RssFeeder#227]
**Message** [View Event](http://seq:5341/#/events?filter=@Id%20%3D%20'event-093cb4bac98508d96c671b0100000000')
```
Response status code = "Forbidden", "https://www.realclearpolitics.com/video/2021/12/27/president_biden_there_is_no_federal_solution_to_covid_test_shortage.html"
```

**Properties**
```
* statusCode : "Forbidden"
* uri : "https://www.realclearpolitics.com/video/2021/12/27/president_biden_there_is_no_federal_solution_to_covid_test_shortage.html"
* urlHash : "b5ca5bbcb1266bf3689ec35033653a74"
* url : "https://t.co/6IuKDr4z8a"
* collectionName : "bad-blue"
```

**Comment by mgnoonan** [comment]
Related to #247

**Comment by mgnoonan** [comment]
Closed by #246
